tests/unit/helpers/validate-properties-fail-test.js

```diff
@@ -0,0 +1,61 @@
+import { moduleFor } from 'ember-qunit';
```

Comment:
@danmcclain I'm OK with that but this file name is odd. If there are tests specific to how this controller is being used the file name needs to reflect that rather than say "fail test"